genhub/genomes/Amel.yml

```diff
@@ -6,14 +6,10 @@ Amel:
         - honeybee
     source: refseq
     branch: invertebrate
-    accession: GCF_000002195.4
-    build: Amel_4.5
+    accession: GCF_003254395.2
+    build: Amel_HAv3.1
```

Comment:
We already have multiple versions of the *Apis mellifera* genome in GenHub, so ideally we'd add a new recipe for the HAv3.1 assembly and annotation rather than replacing the 4.5 version from Baylor. However, since 4.5 is no longer the RefSeq reference, this will require us to adjust the existing recipe to access data from GenBank. There are a couple of recipes in the `genhub/genomes/` directory that already demonstrate this: Znev for instance. Replace `source: refseq` with `source: genbank` and replace the `GCF_` RefSeq identifier with the corresponding `GCA_` GenBank identifier.

Reply:
I think for Amel, we do want to have all three versions documented in yml files because this is going to be a prime example in the manuscript, comparing gene models across assembly/annotation pairs.

When a species name has changed or there is no real advantage to analyzing older data, then let's update and see what we can use in the manuscript. If Daniel want's to preserve original data for his thesis, then versioning of sorts would be best. But as the current plan is to do away with genhub, then up to date data associated with the manuscript will do.